guides/07_FAQs.md

```diff
@@ -242,6 +242,14 @@ const meetingSession = new DefaultMeetingSession(
 );
 ```
 
+### What should my application do in response to the status codes in `audioVideoDidStop`
+
+These status codes can be used for logging, debugging, and possibly notification of end users, but in most cases should not  be used for any retry behavior, as the audio video controller will already be retrying non-terminal errors (i.e. regardless of `MeetingSessionStatus.isTerminal`, your application should not try to immediately restart or recreate the audio video controller).
+
+If `MeetingSessionStatus.isTerminal` returns `true`, you should remove any meeting UX in addition to notifying the user, as the audio video controller will not be retrying the connection.
+
+See the documentation for `MeetingSessionStatusCode` for explanation of the values when used for `audioVideoDidStop`, which may be used to provide more detail when notifying end users, though more general failure messages are recommended unless otherwise noted.
```

Comment:
Need to add link of the documentation to `MeetingSessionStatusCode`.

Reply:
Added.